ci/indy-pool_old.dockerfile

```diff
@@ -0,0 +1,97 @@
+# This dockerfile originates from indy-sdk: https://github.com/hyperledger/indy-sdk/blob/8d4917fbe1c3e49c8bb42b21e15915c5f5459c30/ci/indy-pool.dockerfile
```

Comment:
Just delete this, version control can be used to get it back.

Reply:
This line was part of the original dockerfile from the indy-sdk repository, are you sure this line should be deleted?